docs/setup.md

```diff
@@ -7,6 +7,8 @@ docker-compose up
 open http://localhost:3000
 ```
 
+When docker is running on virtualbox the `3000` port forwarding should be configured.
```

Comment:
idk what this means ... can you elaborate and combine with the `When running on virtual`  below ?

Reply:
Looks better now?